mulesoft.properties

```diff
@@ -0,0 +1,17 @@
+category=Coverage
+description=Coverage analyzer for Mulesoft Workflows
+homepageUrl=https://github.com/UltimateSoftware/sonar-mulesoft
+devVersion=
+name=Mulesoft
+publicVersions=0.5.1
+archivedVersions=
+privateVersions=
+
+defaults.mavenGroupId=com.ultimatesoftware.sonar.plugins.mulesoft
+defaults.mavenArtifactId=mulesoftplugin
+
+0.5.1.description=Initial release, includes basic support for analyzing Mulesoft test coverage reports.
+0.5.1.sqlVersions=[7.3,*]
+0.5.1.downloadUrl=https://search.maven.org/remotecontent?filepath=com/ultimatesoftware/sonar/plugins/mulesoft/mulesoftplugin/0.5.1-RELEASE/mulesoftplugin-0.5.1-RELEASE.jar
+0.5.1.date=2019-09-30
+0.5.1.changelogUrl=https://github.com/UltimateSoftware/sonar-mulesoft#release-051
```

Comment:
GH seems to want a newline at the end here.

Reply:
Added the new line. 🤦